src/gpodder/plugins/soundcloud.py

```diff
@@ -153,8 +165,9 @@ def get_tracks(self, feed):
                 'url': url,
                 'file_size': int(filesize),
                 'mime_type': filetype,
-                'guid': track.get('permalink', track.get('id')),
+                'guid': track_guid,
                 'published': soundcloud_parsedate(track.get('created_at', None)),
+                'total_time': int(track.get('duration') / 1000)
```

Comment:
Another style nitpick: Add the trailing `,` to newly-added lines, so that the diff for the next addition will be nicer (as you see here, the `'published'` line isn't affected by this diff, as the `,` was already there ;).

Reply:
LOL I had been looking at all these trailing commas wondering what the reason was, I'm guessing that will only work with python and maybe some scripting languages on C(++) that would for sure blow up in my face.

Reply:
The C/C++ grammar actually allows for trailing commas in most places (this might not be true for older versions of the standard, but anything >= C++11 should be fine).

Only exception being that Python even allows trailing commas in parameter lists and function calls (and C++ doesn't), but that's super weird and not sure if there's a good reason for it:

    def foo(a, b, c,): ...

    foo(1,2,3,)